cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -60,12 +61,13 @@ damp_trunc = gen.add_group("Damping and Truncation", "damping_truncation")
 damp_trunc.add("numerical_filtering",         bool_t,   0, "Numerical Filtering yes/no",  False)
 damp_trunc.add("damping_method",              int_t,    0, "The damping method to use.", 2, None, None, edit_method=damping_method_enum)
 damp_trunc.add("damping_factor",     double_t, 0, "The constant damping_factor (used in CONSTANT)",  0.01, 0, 1)
-damp_trunc.add("lambda_max",         double_t, 0, "Value for maximum damping_factor (used in MANIPULABILITY/LSV)",  0.1, 0, 10)
+damp_trunc.add("lambda_max",         double_t, 0, "Value for maximum damping_factor (used in MANIPULABILITY/LSV/SIGMOID)",  0.1, 0, 10)
 damp_trunc.add("w_threshold",        double_t, 0, "Value for manipulability threshold (used in MANIPULABILITY)",  0.005, 0, 1)
 damp_trunc.add("beta",               double_t, 0, "Beta for Low Isotropic Damping",  0.005, 0, 1)
+damp_trunc.add("slope_damping",      double_t, 0, "Value for the activation gradient (used in SIGMOID)",  0.003, 0.001,   1)
 damp_trunc.add("eps_damping",        double_t, 0, "Value for least singular value damping",  0.003, 0,   1)
 
-damp_trunc.add("eps_truncation",     double_t, 0, "Value for singular value threshold (used for truncation: sing. value < eps)",    0.001, 0,   1)
+damp_trunc.add("eps_truncation",     double_t, 0, "Value for singular value threshold (used for truncation: sing. value < eps)(used in SIGMOID)",    0.001, 0,   1)
```

Comment:
Please use ```eps_damping``` rather than ```eps_truncation``` as truncation is used in a context that has nothing to do with damping, i.e. it is used during SVD to remove the influence of a singular value that is "close" to 0.0